recipes/full_finetune_distributed.py

```diff
@@ -590,7 +582,7 @@ def recipe_main(cfg: DictConfig) -> None:
             "Distributed finetune recipe should be run via a distributed launcher."
             "If using tune CLI, please specify --nnodes 1 and --nproc_per_node [num_gpus]"
         )
-
+    os.environ["TORCH_NCCL_AVOID_RECORD_STREAMS"] = "1"
     init_process_group(backend="gloo" if cfg.device == "cpu" else "nccl")
```

Comment:
I know this is old, but technically I think we only want nccl if device is cuda. The default for init_process_group just does both which works really well.

Reply:
What other devices do we support right now though?

> The default for init_process_group just does both which works really well

Not sure I understand this.. do you mean we should just be calling `init_process_group()`?